lib/metasploit/framework/data_service/proxy/core.rb

```diff
@@ -122,6 +122,23 @@ def log_error(exception, ui_message)
     raise Exception, "#{ui_message}: #{exception.message}. See log for more details."
   end
 
+  def add_opts_workspace(opts, wspace = nil)
+    opts[:workspace] = wspace if wspace
+
+    # Some methods use the key :wspace. Let's standardize on :workspace and clean it up here.
+    opts[:workspace] = opts.delete(:wspace) unless opts[:wspace].nil?
```

Comment:
Semantically should the `add_opts_workspace` method allow a `:wspace` value in the `opts` parameter to take precedence over the `wspace` parameter?